tap_google_ads/streams.py

```diff
@@ -512,24 +528,28 @@ def sync(self, sdk_client, customer, stream, config, state, query_limit): # pyli
             state['bookmarks'].pop(stream["tap_stream_id"])
             singer.write_state(state)
 
-    def add_account_name(self, customer, record):
-        """Add account name to the record"""
+    def add_account_info(self, customer, record):
+        """Add account name and ID to the record"""
 
+        record["Account ID"] = customer["customerId"]
```

Comment:
:+1:

Reply:
It turns out that that it is enough to have columns provided by the tap as `key_properties` in SCHEMA messages.
So Account ID is not required and I rolled back my change.

Reply:
Makes sense